### PR TITLE
fix soap encoding error

### PR DESCRIPTION
**Pull request by yrong**

### CODE REVIEW

**Comment by jarlah** [comment]
This has worked before. Has the standard always been soap-env etc? Is this a change that is compatible with most legacy devices? Has it been tested against easycwmp just for a start?

File: tr069/src/main/java/com/github/freeacs/tr069/methods/decision/EmptyDecisionStrategy.java [hunk 1]
```diff
@@ -120,6 +121,15 @@ private void writeSystemParameters(
                 toDB.add(ifmp.getCpeParams().get(ifmp.UDP_CONNECTION_URL));
             }
         }
+        CPEParameters cpep = sessionData.getCpeParameters();
```
Comment (jarlah) [inline tr069/src/main/java/com/github/freeacs/tr069/methods/decision/EmptyDecisionStrategy.java]: It sort of looks reasonable and in line with existing code.

**Comment by jarlah** [comment]
The build error is Just a docker secret issue.

**Comment by yrong** [comment]
> This has worked before. Has the standard always been soap-env etc? Is this a change that is compatible with most legacy devices? Has it been tested against easycwmp just for a start?

> I remember the initial code does not work with my Huawei 820 cpe device which does not accept the soap envelop so I did some fix.  for which xmlns attribute I think it is not important but we should keep it consistent everyone in the code. And the effective fix I think is this one:

` tr069/src/main/java/com/github/freeacs/tr069/xml/Response.java
   + sb.append("<?").append("xml").append(" version=\"1.0\" encoding=\"UTF-8\"").append("?>\n");
`

**Comment by jarlah** [comment]
@yrong sorry about not answering for ages. About the encoding part, does it work for more than one type of device? I am not interested in merging this if I am not confident that most devices will handle this change. yes its a simple namespace change, but yey there are lot of badly implemented tr069 clients out there ...

**Comment by jarlah** [comment]
I don't have to time to evaluate this for now. I can't just merge it without knowing why I should

**Pull request closed without merging**